hosts/ponkila-ephemeral-beta/default.nix

```diff
@@ -148,5 +148,10 @@ in
       }
     ];
   };
+
+  # enable rpcbind, redistributable firmware 
+  services.rpcbind.enable = true;
+  hardware.enableRedistributableFirmware = true;
```

Comment:
`hardware.enableRedistributableFirmware` actually I have changed my mind on this, this should be removed. This stuff mainly pulls WiFi drivers etc.